Reflection/ReflectionType.php

```diff
@@ -39,7 +39,7 @@ public function isBuiltin(): bool
      * @link https://php.net/manual/en/reflectiontype.tostring.php
      * @return string Returns the type of the parameter.
      * @since 7.0
-     * @deprecated Since PHP 7.1. Please use {@see ReflectionType::getName()} instead.
+     * @deprecated 7.1. Please use {@see ReflectionType::getName()} instead.
```

Comment:
Could you please remove trailing dot in `7.1.` as it's currently parsed by tests as wrong version. We will update tests to properly process such cases but for now let's just remove the dot.

Reply:
Done https://github.com/JetBrains/phpstorm-stubs/pull/857/commits/d4ecbfd63e3ab81974a8a5ee35cfb04ec0ea1815